### ISSUE TITLE
Branch clean-up

### ISSUE THREAD

**Issue by kdahlquist** [dondi/GRNsight#251]
1.  @anuvarsh and @mihirsamdarshi and @dondi , in the comments to this issue: state what is in each of these branches that is different than master:

        Public

        sliders

        smart-edges

        beta

        control-points

        ga-integration

        mihir's-branch

         refactoring

         spreadsheet controller

         test-suite

2.  Once this has been stated, on a branch by branch basis, and sequentially, we will consolidate the changes in that branch into beta. We will do this one branch at a time, so if we are working on "test-suite", then we do not start any new branches or work on any of the other branches. To validate the consolidation of each branch, we will supply unit tests. We will start with the branch with the smallest number of changes and work our way up.

3.  At the end of this we want to do a release and close off version 1 before beginning new development on version 2.

**Comment by dondi** [comment]
Two branches whose purposes I remember are as follows:
- `control-points`: This branch implemented the transition of the edge curves from circle segments to Bezier curves.
- `ga-integration`: This branch implemented the Google Analytics integration code.

I did not have a clear recollection of the purposes for these branches, but when compared to `master` by GitHub, they were indicated as having all commits in that branch already integrated into `master` (functionality found by choosing a branch then clicking *Compare* on the right-hand side of the tool bar). In some cases, the last commit was made months or even years ago:
- `Public`
- `Sliders`
- `smart-edges`
- `spreadsheet-controller`

Because all of these branches showed full integration to master, I went ahead and deleted all of them.
